src/dev/flang/parser/Lexer.java

```diff
@@ -2518,7 +2518,14 @@ else if (
                     }
                   if (c < 0)
                     {
-                      Errors.unknownEscapedChar(sourcePos(), p, escapeChars);
+                      if (p == (int) '\n')
```

Comment:
You could put \\n in in the `escapeChars` array, but additionally I would think that this escaping should only work for multiline strings, not single line strings? Furthermore we need to support `\r\n`.

Reply:
I think this could be allowed for normal (non-multiline) strings. 

Letting `\` at EOL skip \r\n would be nice. But I am not happy about the added complexity...